rules/windows/process_creation/win_office_spawn_exe_from_users_directory.yml

```diff
@@ -19,13 +19,13 @@ logsource:
     product: windows
 detection:
     selection:
-        ParentImage:
-            - '*\WINWORD.EXE'
-            - '*\EXCEL.EXE'
-            - '*\POWERPNT.exe'
-            - '*\MSPUB.exe'
-            - '*\VISIO.exe'
-            - '*\OUTLOOK.EXE'
+        ParentImage|endswith:
+            - '\WINWORD.EXE'
+            - '\EXCEL.EXE'
+            - '\POWERPNT.exe'
+            - '\MSPUB.exe'
+            - '\VISIO.exe'
+            - '\OUTLOOK.EXE'
```

Comment:
similar thing here with the asterisk (line 30)

Reply:
Fixed